quipucords/client/assets/css/entitlements.css

```diff
@@ -0,0 +1,251 @@
+/**
```

Comment:
Looks like additional files have been added under ```quipucords/client/*``` that should probably be removed if we're aiming for an on-the-fly type of product compile... #764 ... though this particular file does align partially with that concept because of the login templates. I'm partial to removing these files since they'll be updated as we update the UI build